server/server.go

```diff
@@ -294,6 +300,42 @@ func (cs *cribbageServer) ginGetPlayer(c *gin.Context) {
 	c.JSON(http.StatusOK, resp)
 }
 
+// GET /player/:username/allgames
+func (cs *cribbageServer) ginGetActiveGamesForPlayer(c *gin.Context) {
+	pID := model.PlayerID(c.Param(`username`))
+
+	ctx := context.Background()
+	db, err := cs.dbFactory.New(ctx)
+	if err != nil {
+		c.String(http.StatusInternalServerError, `dbFactory.New() error: %s`, err)
+		return
+	}
+	defer db.Close()
+
+	p, err := getPlayer(ctx, db, pID)
+	if err != nil {
+		if err == persistence.ErrPlayerNotFound {
+			c.String(http.StatusNotFound, `Player not found`)
+			return
+		}
+		c.String(http.StatusInternalServerError, `Error: %s`, err)
+		return
+	}
+	games := make(map[model.GameID]model.Game, len(p.Games))
+	for gID := range p.Games {
+		mg, err := getGame(ctx, db, gID)
+		if err != nil {
+			c.String(http.StatusInternalServerError, `Error: %s`, err)
+			return
+		} else if mg.IsOver() {
+			continue
```

Comment:
Instead of this, we should update the SQL to only return games that are not over 🤷